jonathontoon/manifest
# Remove spellcheck attribute to textareas

## Description


Removing spellcheck attribute will enable macOS text replacement feature to textareas in Safari.

## Related Issue





Fixes #122 

## How Has This Been Tested?




- `yarn lint`
- `yarn dev`: compilation successful
- `yarn build`: compilation successful

1. Create a note
2. Try to use a text replacement
3. New text appears

## Types of changes


## Checklist:
